sdk/data/aztables/doc.go

```diff
@@ -0,0 +1,206 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright 2017 Microsoft Corporation. All rights reserved.
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
+
+/*
+
+Package aztables can access an Azure Storage or CosmosDB account.
```

Comment:
Is this essentially a copy of content in the README?

Reply:
It's similar but a much more trimmed down version. I'm not sure what we want to put in the README and in the doc.go so there's a lot of duplicated information

Reply:
Something we can discuss next week.